examples/multiple_windows/lib/app/tooltip_window_content.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: invalid_use_of_internal_member
+// ignore_for_file: implementation_imports
+
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/_window.dart';
+
+class TooltipWindowContent extends StatefulWidget {
+  /// Creates a tooltip window widget.
+  /// [controller] the controller for this window
+  /// [child] the content to render into this window
+  /// [key] the key for this widget
```

Comment:
I know these docs are not super critical inside an example, but typically I would separate each of these lines into its own paragraph with an empty `///` line above and below.